hash/msx2_flop.xml

```diff
@@ -13734,6 +13898,32 @@ The following floppies came with the machines.
 		</part>
 	</software>
 
+	<software name="tokispg">
+		<description>Tokimeki Sports Gal (Japan)</description>
+		<year>1988</year>
+		<publisher>Adult Inn</publisher>
+		<info name="alt_tile" value="ときめきスポーツギャル"/>
+		<info name="usage" value="Requires a Japanese system."/>
+		<part name="flop1" interface="floppy_3_5">
+			<dataarea name="flop" size="737280">
+				<rom name="tokimeki sports gal - adult inn.dsk" size="737280" crc="a70b69b6" sha1="7f02a549ed4770b2ec912cb9d7049c16c1e144cf"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="tominaga">
+		<description>Tominaga Koukou Tantei-Bu (Japan)</description>
+		<year>1993</year>
+		<publisher>Team Plankton</publisher>
+		<info name="alt_tile" value="富永高校探偵部"/>
```

Comment:
If you’re going to be consistent with the other transliterated titles, it should be “Tominaga Koukou Tantei-bu”.

Reply:
Updated to use Tantei-bu